js/components/helpPopover.js

```diff
@@ -34,6 +34,7 @@ export default class HelpPopover extends React.Component {
 
 export class BootstrapPopoverButton extends React.Component {
     popoverName = 'base';
+    popoverTitle = '';
```

Comment:
How does popoverTitle get updated?  Should it be set to accept a props value?

Reply:
This is the base class used for multiple popovers. None of the other instances we already used have a title, like the pin popover. So I just added this option so that in the Import Indicator popover instance I could specify a title while keeping all other instance title free. You can see it defined in the ImportIndicatorsPopover.js file in the ImportIndicatorsButton class.